src/main/java/gov/usgs/wma/waterdata/FieldVisit.java

```diff
@@ -3,14 +3,15 @@
 import java.util.Objects;
 
 public class FieldVisit {
```

Comment:
Consider removing to just us a String, which will correctly implement equals and hashcode.